x-pack/plugins/maps/public/connected_components/timeslider/timeslider.tsx

```diff
@@ -32,6 +40,9 @@ interface State {
   range: number;
   timeslice: [number, number];
   ticks: EuiRangeTick[];
+  isPopoverOpen: boolean;
```

Comment:
Would you mind moving popover and `isPopoverOpen` and `rangeButtonMsg` state into its own component?

Reply:
Done